utils/build_swift/driver_arguments.py

```diff
@@ -788,6 +788,9 @@ def create_argument_parser():
            help='if the Swift Benchmark Suite is run after building, run N '
                 'iterations with -Onone')
 
+    option('--tsan-libdispatch-test', toggle_true,
+           help='run TSan (compiler-rt) libdispatch tests')
```

Comment:
Can you add to the help message that this re-builds compiler-rt against libdispatch using the just built toolchain and then tests the implementation? It would be good to add in a comment above it why we are doing it this way in case people wonder.